dashboard/app/views/plc/learning_modules/new_learning_resource_for_module.html.haml

```diff
@@ -0,0 +1,37 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+
+%h3{style: 'margin-bottom: 24px;'}
+  Add a resource (Module: #{@learning_module.name})
+
+.new_resource_section{style: 'float: left; width: 33%'}
+  %p
+    = link_to 'New Script', new_script_path, class: 'btn'
```

Comment:
It was intended to take you to the "Resource Create/Edit Page (Script Type)", because the assumption is that the script itself is already created. I realize now that "New Script" is misleading. 

Given that we'll be editing the modules/resources in Prod and not Levelbuilder, I think we're better off linking to the Edit Page rather than to a page to create a script - otherwise we'd need to send them to another environment as well.